lib/chef/knife/kvm_base.rb

```diff
@@ -33,6 +33,7 @@ def run_remote_command(command)
               if success
                 channel.on_data do |ch, data|
                   return_val = data.chomp
+                  print return_val if console
```

Comment:
So this is live-streaming the data back from the console through SSH. Since this is raw IO, we can't use ui.* or puts or any of that, so we have to fall back on good old `IO::print`